api/features/versioning/managers.py

```diff
@@ -1,11 +1,12 @@
 import typing
 from pathlib import Path
 
-from django.db.models.query import RawQuerySet
+from django.db.models.query import QuerySet, RawQuerySet
 from softdelete.models import SoftDeleteManager
 
 if typing.TYPE_CHECKING:
     from environments.models import Environment
+    from features.versioning.models import EnvironmentFeatureVersion
```

Comment:
This should be a normal import.

Reply:
No, it can't be. The `models` module imports directly from the managers (and has to).

Reply:
You need an import inside the new interfaces, then.

Reply:
(Have I told you I hate Django btw?)

Reply:
Sorry, my bad, that was just a copypasta error. I've replaced it with `self.get_latest_versions`.